specification/authorization/resource-manager/Microsoft.Authorization/preview/2020-04-01-preview/authorization-RoleAssignmentsCalls.json

```diff
@@ -135,6 +135,12 @@
             "schema": {
               "$ref": "#/definitions/RoleAssignmentListResult"
             }
+          },
+          "400": {
+            "description": "Bad Request - Indicates the request has missing information and can not be completed",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
```

Comment:
Hi @dagoroz @yungezz could any one explain this to me, why we have to add these "error response code" into this list?
In the generated SDKs, the status code listed here will **NOT** be considered as errors.
To indicate errors, please put the errors in the `default` response.

Reply:
Same opinion as @ArcturusZhang. If 400 is expected as error, it should be covered by `default`, not `400`.

This change breaks track2 Java SDK, as `RoleAssignmentListResult` and `Error` is 2 completely different type. Hence the method will not able to figure out which type will be returned in runtime.